app/assets/stylesheets/application.css

```diff
@@ -67,6 +67,10 @@ a {
   height: 10vh;
 }
 
+.text-white {
+  align-items: center;
+}
+
 .bg-none {
   background: none;
   border: none;
```

Comment:
# Design
![design](https://user-images.githubusercontent.com/12745474/221381527-6925be5b-0738-406c-b563-763b08337fb2.png)
# Your Project
![image](https://user-images.githubusercontent.com/93039359/222922467-39a5d75b-77ab-4903-995e-51b65bb3bb04.png)


First of all, great work on implementing some of the changes requested by the previous reviewer :+1: but you missed some of the points he raised actually :thinking:

- [ ] First, it would be nice if you make the heading of the page center such as Transaction should be in the center ho on the right or left side to follow the design :+1:
- [ ] Secondly, the previous reviewer told you to remove the log-out button because it was showing on all the pages so it doesn't make our design look so much attractive and professional but you have removed the logout button from everywhere in your application no we are not able to log out from your application I apologize on the behalf of the previous reviewer he actually made a mistake while writing this change. The thing he wanted you to do was to put your logout button in the header section of your application and remove it from the bottom of every page :smile: Kindly add a button in the header section of your application :+1:
- [ ] Last, the previous reviewer mentioned that your button such as Add Transaction or Add Category should look similar to the Design, and the design button is the bottom which has a green color and white inside border :+1: